packages/react/src/components/cookieConsent/getContent.ts

```diff
@@ -0,0 +1,806 @@
+export function getCookieContent() {
+  const commonLanguageTranslations = {
+    fi: {
+      description: 'Eväste vaaditaan jotta käyttäjän kielivalinta säilyisi.',
+      expiration: 'Istunto',
+    },
+    sv: {
+      description: 'Kakan krävs för att spara användarens språkval.',
+      expiration: 'Session',
+    },
+    en: {
+      description: "Required to persist the user's chosen language..",
```

Comment:
I don't think there's a need for passive aggressive dots at the end of sentence 😆

Reply:
:) Good catch!